DigitalOffice/Tests/UserServiceTests/Validators/UserCreateRequestValidatorTests.cs

```diff
@@ -85,6 +85,11 @@ public void ShouldThrowValidationExceptionWhenPasswordIsEmpty()
             validator.ShouldHaveValidationErrorFor(x => x.Password, "");
         }
 
+        public void ShouldThrowValidationExceptionWhenIsAdminIsEmpty()
```

Comment:
Add Test attribute

Reply:
Fixed.
I've deleted this test because it's extra - bool can't get null at all.